src/app/app.module.ts

```diff
@@ -3,21 +3,29 @@ import { BrowserModule } from '@angular/platform-browser';
 import { RouteReuseStrategy } from '@angular/router';
 
 import { IonicModule, IonicRouteStrategy } from '@ionic/angular';
+import { IonicStorageModule } from '@ionic/storage';
 import { SplashScreen } from '@ionic-native/splash-screen/ngx';
 import { StatusBar } from '@ionic-native/status-bar/ngx';
 
 import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app-routing.module';
-import { ExampleCoreService } from './core/example-service/example-core.service';
+import { ExampleCoreService } from './core/services/example-service/example-core.service';
+import { StorageService } from './core/services/storage/storage.service';
 
 @NgModule({
   declarations: [AppComponent],
   entryComponents: [],
-  imports: [BrowserModule, IonicModule.forRoot(), AppRoutingModule],
+  imports: [
+    BrowserModule,
+    IonicModule.forRoot(),
+    IonicStorageModule.forRoot(),
```

Comment:
Where are you setting IonicStorage as the default storage for Akita? 

If you want to use Ionic as the default, you need to explicitly set it: https://datorama.github.io/akita/docs/enhancers/persist-state

The problem is that IonicStorage does not implement the StorageEngine API, so we would need a wrapper for that :(